selfservice/strategy/link/strategy_recovery.go

```diff
@@ -56,16 +56,20 @@ func (s *Strategy) PopulateRecoveryMethod(r *http.Request, f *recovery.Flow) err
 	return nil
 }
 
-// swagger:parameters adminCreateSelfServiceRecoveryLink
+// Create Recovery Link for Identity Parameters
+//
+// swagger:parameters createRecoveryLinkForIdentity
 //
 // nolint
-type adminCreateSelfServiceRecoveryLink struct {
+type createRecoveryLinkForIdentity struct {
 	// in: body
-	Body adminCreateSelfServiceRecoveryLinkBody
+	Body createRecoveryLinkForIdentityBody
 }
 
-// swagger:model adminCreateSelfServiceRecoveryLinkBody
-type adminCreateSelfServiceRecoveryLinkBody struct {
+// Create Recovery Link for Identity Request Body
+//
+// swagger:model createRecoveryLinkForIdentityBody
+type createRecoveryLinkForIdentityBody struct {
```

Comment:
Do we have (or plan to have) recovery links for something else than an identity? 

I'd prefer `createRecoveryLink`, because it's more concise and contains the same amount of information.

Reply:
Hm, not really. 

Maybe we should keep the `forIdentity` as this is the admin side of things, and that's what it does: it creates a recovery link for an identity.

So with the potential for a `frontend.createRecoveryLink(forMe)`, let's keep it.